doc/source/user_guide/convert.rst

```diff
@@ -345,465 +344,462 @@ using the :attr:`Mapdl.non_interactive <ansys.mapdl.core.Mapdl.non_interactive>`
-    NSEL,R,LOC,Z,0
-    /OUT,SCRATCH
-    FSUM                                ! FZ IS TOTAL LOAD FOR PIPE288 MODEL
-    *GET,LOAD_288,FSUM,,ITEM,FZ
-    NSEL,S,NODE,,101,118                ! SELECT NODES IN SOLID185 MODEL
-    NSEL,R,LOC,Z,0
-    FSUM
-    *GET,ZFRC,FSUM,0,ITEM,FZ
-    LOAD=ZFRC*360/THETA                 ! MULTIPLY BY 360/THETA FOR FULL 360 DEGREE RESULTS
-    *STATUS,LOAD
-    LOAD_185 = LOAD
-    NSEL,S,NODE,,201,212                ! SELECT NODES IN SHELL181 MODEL
-    NSEL,R,LOC,Z,0
-    FSUM
-    /OUT,
-    *GET,ZFRC,FSUM,0,ITEM,FZ
-    LOAD=ZFRC*360/THETA                 ! MULTIPLY BY 360/THETA FOR FULL 360 DEGREE RESULTS
-    *STATUS,LOAD
-    LOAD_181 = LOAD
-    *VFILL,VALUE_288(1,1),DATA,1024400,1262000,1262000
-    *VFILL,VALUE_288(I,2),DATA,ABS(LOAD_288)
-    *VFILL,VALUE_288(I,3),DATA,ABS(LOAD_288)/(VALUE_288(I,1))
-    *VFILL,VALUE_185(1,1),DATA,1024400,1262000,1262000
-    *VFILL,VALUE_185(J,2),DATA,ABS(LOAD_185)
-    *VFILL,VALUE_185(J,3),DATA,ABS(LOAD_185)/(VALUE_185(J,1))
-    *VFILL,VALUE_181(1,1),DATA,1024400,1262000,1262000
-    *VFILL,VALUE_181(K,2),DATA,ABS(LOAD_181)
-    *VFILL,VALUE_181(K,3),DATA,ABS(LOAD_181)/(VALUE_181(K,1))
-    *END
-    /COM,' GET TOTAL LOAD FOR DISPLACEMENT = 0.032'
-    /COM,' ---------------------------------------'
-    SET,1,1
-    I = 1
-    J = 1
-    K = 1
-    *DIM,LABEL,CHAR,3,2
-    *DIM,VALUE_288,,3,3
-    *DIM,VALUE_185,,3,3
-    *DIM,VALUE_181,,3,3
-    *USE,GETLOAD
-    /COM,' GET TOTAL LOAD FOR DISPLACEMENT = 0.05'
-    /COM,' --------------------------------------'
-    SET,2,1
-    I = I + 1
-    J = J + 1
-    K = K + 1
-    *USE,GETLOAD
-    /COM,' GET TOTAL LOAD FOR DISPLACEMENT = 0.1'
-    /COM,' -------------------------------------'
-    SET,3,1
-    I = I +1
-    J = J + 1
-    K = K + 1
-    *USE,GETLOAD
-    LABEL(1,1) = 'LOAD, lb','LOAD, lb','LOAD, lb'
-    LABEL(1,2) = ' UX=.032',' UX=0.05',' UX=0.10'
-    FINISH
-    /OUT,vm7,vrt
-    /COM,------------------- VM7 RESULTS COMPARISON ---------------------
-    /COM,
-    /COM,'                 |   TARGET   |   Mechanical APDL   |   RATIO'
-    /COM,
-    /COM,RESULTS FOR PIPE288:
-    /COM,
-    *VWRITE,LABEL(1,1),LABEL(1,2),VALUE_288(1,1),VALUE_288(1,2),VALUE_288(1,3)
-    (1X,A8,A8,'   ',F10.0,'  ',F14.0,'   ',1F15.3)
-    /COM,
-    /COM,RESULTS FOR SOLID185:
-    /COM,
-    *VWRITE,LABEL(1,1),LABEL(1,2),VALUE_185(1,1),VALUE_185(1,2),VALUE_185(1,3)
-    (1X,A8,A8,'   ',F10.0,'  ',F14.0,'   ',1F15.3)
-    /COM,
-    /COM,RESULTS FOR SHELL181:
-    /COM,
-    *VWRITE,LABEL(1,1),LABEL(1,2),VALUE_181(1,1),VALUE_181(1,2),VALUE_181(1,3)
-    (1X,A8,A8,'   ',F10.0,'  ',F14.0,'   ',1F15.3)
-    /COM,
-    /COM,-----------------------------------------------------------------
-    /OUT
-    *LIST,vm7,vrt
-
-Convert the verification file with:
-
-.. code:: pycon
-
-    >>> from ansys.mapdl import core as pymapdl
-    >>> pymapdl.convert_script("vm7.dat", "vm7.py")
-
-Here is the translated Python script:
```

Comment:
I think this part has been totally removed. Can you put it somewhere? Maybe before the tabs?

Reply:
It has not been removed, it has been modified [here](https://github.com/ansys/pymapdl/pull/2237/files#diff-42a6c55f6287c841d60c446ea8eafac6fc451a8d205fbbee72db4a7825e45171R347).

Reply:
The link takes to the files changed tab. But if you are sure this has not been removed, you can mark this as resolved.